### DISCUSSION TITLE
Placeholder text for entry not showing

### DISCUSSION THREAD

**Discussion by nawrazaltai** [TomSchimansky/CustomTkinter#286] [Q&A] [answered]
Hi guys, I'm kind of a beginner so please bare with me..

Maybe I'm missing something obvious but when using a text_variable in CTkEntry my placeholder_text doesn't show in the entrybox. When removing the text_variable then the placeholder_text works fine. Any ideas..?

**Comment by TomSchimansky** [comment, marked as answer]
The placeholder_text doesn't work with a variable. You can just either use a variable or a placeholder_text. That's because it was too difficult to implement, there a so many cases that don't work and where its unclear what should happen with the placeholder if you also use a variable, so I deactivated the placeholder if a variable is set.

**Reply by nawrazaltai** [reply]
I get it, thank you for the answer and for CustomTkinter.
